### ISSUE TITLE
Clean `TODO` comments in the code base and transform them into issues

### ISSUE THREAD

**Issue by AucaCoyan** [federico-terzi/espanso#1938]
It's better to have all the task combined in one place: GitHub issues 😄 

We can also tag some of them with `chores` or something similar to indicate that are small tasks

**Comment by rbstrachan** [comment]
This isn't a bad idea, although a cursory search found 127 instances of `TODO` in 78 files. Some simply say "test this" and others seem to be from 2021–2022 so are quite old and may no longer be relevant. I'll have a look to see if I can find some useful comments to be converted to issues.

**Comment by AucaCoyan** [comment]
Yeah, a lot of those probably aren't even relevant. The most important thing for me is that those are "issues hidden away", and should be bring upfront to know what we are missing and prioritize everything on the table [project](https://github.com/orgs/espanso/projects/1)
